atmcfg/atmcfg_test.go

```diff
@@ -62,6 +63,130 @@ func TestStartup(t *testing.T) {
 	})
 }
 
+func TestShutdownProcessesByClusterName(t *testing.T) {
+	t.Run("replica set", func(t *testing.T) {
+		config := automationConfigWithOneReplicaSet(clusterName, false)
+		_ = ShutdownProcessesByClusterName(config, clusterName, []string{"host0:27017"})
```

Comment:
👀  please, check no error occurs, don't ignore because if this start throwing an error we won't know and the test will be useless